accessibility/demo/src/main/java/com/novoda/accessibility/demo/custom_actions/TweetView.java

```diff
@@ -0,0 +1,122 @@
+package com.novoda.accessibility.demo.custom_actions;
+
+import android.content.Context;
+import android.support.v4.view.ViewCompat;
+import android.util.AttributeSet;
+import android.view.View;
+import android.widget.Button;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import com.novoda.accessibility.AccessibilityServices;
+import com.novoda.accessibility.Action;
+import com.novoda.accessibility.Actions;
+import com.novoda.accessibility.ActionsAccessibilityDelegate;
+import com.novoda.accessibility.ActionsAlertDialogCreator;
+import com.novoda.accessibility.demo.R;
+
+import java.util.Arrays;
+
+public class TweetView extends LinearLayout {
+
+    private TextView tweetTextView;
+    private Button replyButton;
+    private Button retweetButton;
+    private AccessibilityServices services;
+
+    public TweetView(Context context, AttributeSet attrs) {
+        super(context, attrs);
+        setOrientation(VERTICAL);
```

Comment:
yep would do in the general case, wanted to keep the demo as simple as possible - the aim is not to showcase our internal conventions, but rather highlight the usage of the lib (cc @mr-archano )